Runtime/Scripts/RGBotConfigs/RGState.cs

```diff
@@ -43,6 +47,7 @@ public class RGState: MonoBehaviour
             ["id"] = this.transform.GetInstanceID(),
             ["type"] = objectType,
             ["isPlayer"] = isPlayer,
+            ["isStatic"] = isStatic,
```

Comment:
Maybe we name this `isSceneElement` or `isDynamicallySpawned` ?

Would those names better represent that this is a thing that isn't defined in the scene itself, but it dynamically loaded at runtime based on the game flow , player connection, game event, etc.

Reply:
@rcornwall-reg any idea on what developers call this? GameObjects that are dragged into the scene within the editor vs spawned in after the scene starts?